webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -35,31 +37,53 @@ const TrialChart: React.FC<Props> = ({
   metricNames,
   metrics,
   onMetricChange,
-  workloads,
   trialId,
+  trialTerminated,
 }: Props) => {
   const [ scale, setScale ] = useState<Scale>(Scale.Linear);
+  const [ trialSumm, setTrialSummary ] = useState<MetricContainer[]>([]);
+
+  const fetchTrialSummary = useCallback(async () => {
+    if (trialId) {
+      const summ = await compareTrials({
+        maxDatapoints: screen.width > 1600 ? 1500 : 1000,
+        metricNames: metricNames,
+        scale: scale,
+        startBatches: 0,
+        trialIds: [ trialId ],
+      });
+      setTrialSummary(summ[0].metrics);
+    }
+  }, [ metricNames, scale, trialId ]);
+
+  const { stopPolling } = usePolling(fetchTrialSummary, { interval: 2000, rerunOnNewFn: true });
+  useEffect(() => {
+    if (trialTerminated) {
+      stopPolling();
+    }
+  }, [ trialTerminated, stopPolling ]);
+  if (trialTerminated) {
+    stopPolling();
```

Comment:
so i guess this extra one is needed in case you start polling when the trial is in terminal state?